build/Build.cs

```diff
@@ -189,6 +189,33 @@ class Build : NukeBuild
         });
     });
 
+    Target PackMergedClientNugetForMacOs => _ => _
+     .OnlyWhenStatic(() => IsLocalBuild)
+     .OnlyWhenStatic(() => OperatingSystem.IsMacOS())
```

Comment:
You could merge `.OnlyWhenStatic(() => IsLocalBuild && !OperatingSystem.IsWindows())`

Reply:
As per your comments below, I've remove the operating system check altogether 👍